webview/src/plots/components/styles.module.scss

```diff
@@ -168,13 +231,13 @@ $gap: 20px;
 .plot.multiViewPlot {
   aspect-ratio: calc(0.8 * var(--scale) + 0.2);
   grid-column: span var(--scale);
-  width: calc(var(--size) * var(--scale));
+  width: calc(300px * var(--scale));
```

Comment:
[Q] Is this intended behaviour: 

https://user-images.githubusercontent.com/37993418/203680633-346f9205-c3ce-483e-9f8b-f61a42e986cd.mov

Should it be possible to resize a multiview plot? Should the option be removed from multiview plots?

Reply:
No it shouldn't be possible (yet). Forgot to remove it.